include/mqtt.h

```diff
@@ -181,6 +182,14 @@ int __cheri_compartment("MQTT") mqtt_publish(Timeout    *t,
  *
  * `qos` indicates the level of QoS (0, 1, or 2).
  *
+ * `retain` indicates whether the message should be published as retained or
```

Comment:
I suspect that this should be added to the documentation of `mqtt_publish`, not `mqtt_subscribe`?

Reply:
Doh !